QuickStart/ChatViewController.swift

```diff
@@ -81,6 +117,14 @@ class DeskChannelModule {
                 
                 return cell
             }
+            // When the message is inquire ticket closure
+            else if let userMessage = message as? UserMessage, userMessage.data.contains("\"type\":\"SENDBIRD_DESK_INQUIRE_TICKET_CLOSURE\""), userMessage.data.contains("\"state\":\"WAITING\"") {
```

Comment:
Since this is in `cellForRow`, wouldn't the confirmation alert show whenever the user scrolls away/to this message?

Reply:
Yes your're right. I'll modify to present alert when it receives the new message via view model delegate

Reply:
[1a43d60](https://github.com/sendbird/quickstart-desk-ios/pull/14/commits/1a43d60fcaa0d744d01fa5d11329b71e05d2a8c5)